code/game/machinery/cryopod.dm

```diff
@@ -228,9 +228,14 @@
 		/obj/item/autopsy_scanner,
 		/obj/item/holosign_creator/atmos,
 		/obj/item/clothing/gloves/color/black/forensics,
-		/obj/item/rcd,
-		/obj/item/rpd,
-		/obj/item/mod/control
+		/obj/item/mod/control,
+		/obj/item/stamp,
+		/obj/item/melee/knuckleduster,
```

Comment:
Wouldn't this be /all/ dusters? not just the gold one?

Reply:
Yes actually, fixed